db/sql/5_read_only_user.sql

```diff
@@ -0,0 +1,4 @@
+CREATE USER reader WITH LOGIN PASSWORD 'acab';
```

Comment:
Since we can play with order all we want, I wonder if these two should be `2_` and `3_` instead of `5_` and `6_` respectively. Since this is more database setup than dataset creation.

Reply:
IIRC I had to put the user and scoping at the end but I don't remember why exactly.